src/thebelab.js

```diff
@@ -426,7 +427,7 @@ export function requestBinder({
           es.close();
           resolve(
             ServerConnection.makeSettings({
-              baseUrl: msg.url,
+              baseUrl: msg.url  + "/tree/" + defaults.workingDir,
```

Comment:
The base URL here needs to be the base URL of the REST API. Where the kernel starts is not connected to this value.